src/database/standalone-sqlite.ts

```diff
@@ -309,7 +309,7 @@ export function dataItemToDbRows(item: NormalizedDataItem, height?: number) {
       target: fromB64Url(item.target),
       data_offset: item.data_offset,
       data_size: item.data_size,
-      content_type: contentType,
+      content_type: contentType ?? item.content_type,
```

Comment:
this allows content type to first come from Content-Type tag if exists, otherwise use from the incoming normalized data item headers if that exists

Reply:
fwiw I found this when there was `null` in content type in my e2e testing. should we also default to application/octet-stream here?

Reply:
Good question, but nah, we lose the ability to distinguish the case when that's being set explicitly vs omitted if we put a default here. We have default on the way out when we serve the data.